tests/resources/test-types.json

```diff
@@ -1096,7 +1123,73 @@
             "linkedTestCode": null
           }
         ]
-      }
+      },
+    {
+      "id": "37",
```

Comment:
When you add or remove test-types here, there should a front-end ticket to reflect these changes in `test-types-fields.metadata.ts`. @vladddragomir am I correct?